brigade-worker/src/events.ts

```diff
@@ -70,6 +65,25 @@ export class BrigadeEvent {
   cause?: Cause;
 }
 
+/**
+ * Revision describes a vcs revision.
+ */
+export class Revision {
+  /**
+   * commit is the upstream VCS commit ID (revision).
+   *
+   * If it is not provided, it may be interpreted as `master`, or the head
+   * of the main branch.
+   *
+   * The default value is not guaranteed to be `master` in future versions.
+   */
+  commit?: string;
```

Comment:
As you describe this in the comment above, I believe this should be `id` rather than `commit`.
A bit more context is that, in my mental model, `revision` almost equal to `commit`. So for example, `revision.id` sounds more natural to me than `revision.commit`.
WDYT?

Reply:
That's a good point. I wonder what the Git documentation calls these.

Reply:
Actually, it is called just "SHA-1" throughtout man-pages and the textbook at git-scm.com as far as I know.

Example: https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection

Based on that and assuming a "revision" has up to 1 git-commit in it, I'm also ok with `revision.sha1` rather than `revision.commit`.

Btw, `revision.ref` makes sense to me, because it refers to a git-ref, which I believe that it is already used widely/officially in git.

WDYT?